9.mAP/notebooks/0.generate_map_scores_morphology.ipynb

```diff
@@ -1,1093 +1,495 @@
 {
```

Comment:
<p>This is more of a clarification comment. When attempting to determine the null value, are we examining the number of samples, specifically the count of pyroptosis samples?&nbsp;</p><p> </p><p>I'm trying to understand on why 65. &nbsp;</p><p> </p><p>We can discuss about this in lab!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Yes, this compares the null distrubiution with the sample size of the pyroptosis group compared to the other groups</p>